packages/trpc/server/routers/viewer/organizations/update.handler.ts

```diff
@@ -80,13 +88,11 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
       data.slug = input.slug;
 
       // If we save slug, we don't need the requestedSlug anymore
-      const metadataParse = teamMetadataSchema.safeParse(prevOrganisation.metadata);
-      if (metadataParse.success) {
-        const { requestedSlug: _, ...cleanMetadata } = metadataParse.data || {};
-        data.metadata = {
-          ...cleanMetadata,
-        };
-      }
+      const { requestedSlug: _, ...cleanMetadata } = prevMetadata || {};
+      data.metadata = {
+        ...cleanMetadata,
+        ...input.metadata,
```

Comment:
Taking into account passed down metadata letting it overwrite existing values

Reply:
We have the same issue here: It will overwrite the metadata when setting requestedSlug:

https://github.com/calcom/cal.com/blob/376f023e0fd69c0f30569152785b56c277a32bc8/packages/trpc/server/routers/viewer/organizations/update.handler.ts#L76-L78